vscode/package.nls.json

```
@@ -65,6 +68,12 @@
     "jdk.debugger.configuration.attach.listen.description": "Listen for the debuggee to attach",
     "jdk.debugger.configuration.attach.timeout.description": "Timeout while waiting to attach",
     "jdk.debugger.configuration.completion.warning.time.description": "When code completion takes longer than this specified time (in milliseconds), there will be a warning produced (-1 to disable)",
+    "jdk.notebook.classpath.description": "Classpath that needs to be set for notebooks",
+    "jdk.notebook.modulepath.description": "Modulepath that needs to be set for notebooks",
+    "jdk.notebook.addmodules.description": "Modules that needs to be added for notebooks",
+    "jdk.notebook.enablePreview.description": "Flag to enable jdk preview features for notebooks",
+    "jdk.notebook.implicitImports.description": "List of packages to import implicitly",
```

```suggestion
    "jdk.notebook.implicitImports.description": "List of elements to implicitly import in Java notebooks. Defaults to star-imports of java.util, java.io and java.math packages, when empty.",
```

An additional `.markdownDescription` label may also be added for prettier formatting.